docs/site-replication/run-multi-site-ldap.sh

```diff
@@ -166,6 +166,35 @@ if [ $? -ne 0 ]; then
     exit_1;
 fi
 
+vID=$(./mc stat minio1/bucket/README.md --json | jq .versionID)
+if [ $? -ne 0 ]; then
+    echo "expecting object to be present. exiting.."
+    exit_1;
+fi
+./mc tag set --version-id "${vID}" minio1/bucket/README.md "k=v"
+if [ $? -ne 0 ]; then
+    echo "expecting tag set to be successful. exiting.."
+    exit_1;
+fi
+sleep 5
+
+./mc tag remove --version-id "${vID}" minio1/bucket/README.md
+if [ $? -ne 0 ]; then
+    echo "expecting tag removal to be successful. exiting.."
+    exit_1;
+fi
+sleep 5
+
+replStatus_minio1=$(./mc stat minio1/bucket/README.md --json | jq .)
+if [ $? -ne 0 ]; then
+    echo "expecting object to be present. exiting.."
+    exit_1;
+fi
+if [ "${replStatus_minio1}" != "COMPLETED" ]; then
+    echo "expected tag removal to have replicated, exiting..."
+    exit_1;
+fi
```

Comment:
We should check the tag status on the destination @poornas isn't that we wanted to validate?